elasticsearch_k8s_monitoring/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "elasticsearch_monitoring_chart_version" {
+  type        = string
+  description = "elasticsearch-monitoring Helm chart version to deploy"
+  default     = "0.2.5"
+}
+
+variable "elasticsearch_endpoint" {
+  type        = string
+  description = "Endpoint of the AWS Elasticsearch domain"
+}
+
+variable "elasticsearch_domain_name" {
+  type        = string
+  description = "Domain name of the AWS Elasticsearch domain"
+}
+
+variable "elasticsearch_domain_region" {
+  type        = string
+  description = "Region of the AWS Elasticsearch domain"
+}
+
+variable "kubernetes_context" {
+  type        = string
+  description = "Kubeconfig context to use for deploying the `skyscrapers/elasticsearch-monitoring` chart"
+}
+
+variable "kubernetes_namespace" {
+  type        = string
+  description = "Kubernetes namespace where to deploy the `skyscrapers/elasticsearch-monitoring` chart"
+}
+
+variable "kubernetes_worker_instance_role_arns" {
+  type        = list
```

Comment:
I think you always have to specify the type of the list elements

Reply:
you don't, as it works :P. Still, good practice to do so